### ISSUE TITLE
Separate browse search from Map search

### ISSUE THREAD

**Issue by Unachieved** [gaskij/rpicampusmap#103]
Both search options do the same thing. Add maps specific search bar --- points out location on the map. All searching outside the map can be linked to browse --- add goto maps option onclick of browse options (rn it just goes to info tabs)

**Comment by gaskij** [comment]
This is a good point, never though about doing this directly. If there are multiple options that match the search query, what do you think would be a good way to handle this? Should we try to implement a sidebar similar to google maps?

**Comment by Unachieved** [comment]
Yeah that's sounds good to me. I'm thinking the side bar pops out on the
condition that there are multiple hits for the query. Then the user can
select the one they mean.

Also I'm going to revise the issues using the "CORE, FEATURE, ASIDE"
starting and separating some of them into multiple issues. I'll put a note
of exactly what those mean. Been working on it for a while so I'm taking a
break and getting my other stuff out of the way first.

I also updated the read me a bit and have some announcements to make about
documentation.
